charts/kubescape-operator/templates/node-agent/daemonset.yaml

```diff
@@ -53,6 +53,19 @@ spec:
       serviceAccountName: {{ .Values.nodeAgent.name }}
       automountServiceAccountToken: true
       hostPID: true
+      {{- if .Values.nodeAgent.initContainer }}
```

Comment:
Please use another name instead the `initContainer` flag. Maybe `startupJitterContainer` or something more descriptive

Reply:
Done